bridge/client/app/_components/ktb-event-item/ktb-event-item.component.html

```diff
@@ -45,6 +45,13 @@ <h4 class="m-0 mt-1 mb-1" [class.error]="_event.isFaulty()" [textContent]="_even
       </dt-tag-list>
     </div>
   </div>
+  <div *ngIf="_event.isApproval() && project.getRootEvent(project.getService(_event.data.service), _event).isApproval()">
```

Comment:
Not necessary to change this in this PR, but in general function calls should be avoided in templates. Angular doesn't cache function results, so these functions are executed with every change detection cycle. This could degrade performance, depending on how often change detection is fired.
https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496
https://stackblitz.com/edit/angular-hxmb6q

Reply:
Thanks @dominikmessner, actually I need to processes that once and store the reference in the event model. We have some refactoring tasks, see #1905, where I could address this issue, since I use this approach now already at several places.